tools/clang/test/hlsl_cmds_test/batch_cmds.txt

```diff
@@ -0,0 +1,5 @@
+-E ps_main -T ps_6_0     lib_entries2.hlsl
```

Comment:
It is helpful to take files that are used as the inputs to tests rather than tests themselves and move them into an `Inputs` subdirectory.